pkg/helper/options/logs_test.go

```diff
@@ -126,6 +132,29 @@ func TestLogOptions_Ask(t *testing.T) {
 			want: LogOptions{
 				PipelineName:    "",
 				PipelineRunName: "sample-pipeline-run1",
+				TaskrunName:     "",
+			},
+		},
+		{
+			name:     "select taskrun name",
+			resource: ResourceNameTaskRun,
+			prompt: htest.PromptTest{
+				CmdArgs: []string{},
+				Procedure: func(c *goexpect.Console) error {
+					if _, err := c.ExpectString("Select taskrun :"); err != nil {
```

Comment:
```suggestion
					if _, err := c.ExpectString("Select taskrun:"); err != nil {
```

Can you remove the space before `:` and also in the associated code in [`Ask`](https://github.com/tektoncd/cli/blob/de4cb226a74c9d790976c7a9f028424b4f173e26/pkg/helper/options/logs.go#L75)?